Client.xcodeproj/project.pbxproj

```diff
@@ -7797,7 +7794,6 @@
 				2C2A5EF41E68469500F02659 /* PrivateBrowsingTest.swift in Sources */,
 				3D71C89E1F5703A1008D8646 /* CopiedLinksTests.swift in Sources */,
 				2CF449A51E7BFE2C00FD7595 /* NavigationTest.swift in Sources */,
-				2C32EA442100C4BF00A25912 /* TabTraySearchTabsTests.swift in Sources */,
```

Comment:
The option to search tabs in the tab tray has been removed. No need to keep this test suite for now